src/RadioButtons/index.js

```diff
@@ -100,7 +100,7 @@ const RadioButtons = ({
                 role="presentation"
                 className={cc([
                   "nds-radio",
-                  { "narmi-icon-check": kind === "card" },
+                  { "narmi-icon-check": ["card", "checkmark"].includes(kind) },
```

Comment:
Had an initial nit, but after double checking some includes behaviors, retracted it! This looks awesome